spec/game_team_spec.rb

```diff
@@ -3,3 +3,46 @@
 RSpec.configure do |config| 
  config.formatter = :documentation 
 end
+
+RSpec.describe GameTeam do 
+  before(:each) do 
+ 
+  @game_teams_data = GameTeam.new({})
+     # game_path = './data/games.csv'
+     # team_path = './data/teams.csv'
+     # game_teams_path = './data/game_teams.csv'
+ 
+     # locations = {
+     #   games: game_path,
+     #   teams: team_path,
+     #   game_teams: game_teams_path
+     # }
+ 
+     # # tracker = StatTracker.from_csv(locations)
+   
+   end
+ 
+   describe '#initialize' do
+     it 'exists' do
+       expect(@game_teams_data).to be_an_instance_of(GameTeam)
+     end
+ 
+     xit 'has attributes' do
```

Comment:
Can this be unskipped onw that you have the dummy stats?

Reply:
Looks like a few other things in the test would need to be adjusted a little first.